_examples/unicode/test.py

```diff
@@ -7,17 +7,30 @@
 ## py2/py3 compat
 from __future__ import print_function, unicode_literals
 
+import sys
+
 import encoding
 
+# There is no portable way of outputting Unicode in Python 2/3 -- we encode
+# them manually and sidestep the builtin encoding machinery
+try: binary_stdout = sys.stdout.buffer  # Python 3
```

Comment:
cosmetics: could you align `binary_stdout` with the one from the line below ?

Reply:
I put the blocks on separate lines as sane people and PEP8 recommend.